plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerMetadata.java

```diff
@@ -501,14 +502,14 @@ private synchronized TableInfo createTableInfoFromStats(SchemaTableName tableNam
                         minimums.get(column.name()),
                         maximums.get(column.name()),
                         requireNonNull(distinctValues.getOrDefault(column.name(), 0L)),
-                        requireNonNull(nonNullValues.getOrDefault(column.name(), -1L)),
+                        Optional.ofNullable(nonNullValues.get(column.name())).map(OptionalLong::of).orElse(OptionalLong.empty()),
```

Comment:
It's fine to use `Optional<Long>` as well, we don't need to make this verbose just to get `OptionalLong`